reunioes/progressos/intermediario/php_oo/pdo/InsertBD.php

```diff
@@ -0,0 +1,17 @@
+<?php
+$con = new PDO("mysql:host=localhost;dbname=aprendendo" , "root", "root");
+
+$stmt = $con->prepare("INSERT INTO usuarios (nome, idade) VALUES(:NAME, :AGE)");
+
+$nome = "Michelle";
+$idade = "25";
```

Comment:
Não inclua aspas quando for trabalhar com números, pois assim você forma como string e pode causar dificuldades em validações posteriores.